templates/includes/sidebar.html

```diff
@@ -1,4 +1,20 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      <a href="{% url 'taxi:driver-detail' user.id %}">Driver: {{ user.first_name }}</a>
+    </li>
+    <li>
+      <a href="{% url 'logout' %}">Log out</a>
```

Comment:
The 'logout' URL is used here. Make sure that the URL pattern for logout is correctly defined in your project's URL configuration.